mysql-test/suite/versioning/t/create.test

```diff
@@ -257,6 +257,23 @@ select y from t2 for system_time all where st = @st and en = @en;
 select st, en from t2 for system_time all where y = 2 into @st, @en;
 select y from t2 for system_time all where st = @st and en = @en;
 
+set default_storage_engine = myisam;
+create or replace table t1 (a int) with system versioning engine innodb;
+create or replace table t2 as select a, sys_trx_start, sys_trx_end from t1 for system_time all;
+--error ER_VERS_FIELD_WRONG_TYPE
+create or replace table t2 engine myisam as select a, sys_trx_start, sys_trx_end from t1 for system_time all;
+
+create or replace table t1 (a int, id int) with system versioning engine innodb;
```

Comment:
Use opposite of default_storage_engine instead of hardcoded `innodb`. See `default_engine()` in `common.inc`.